package.json

```diff
@@ -33,26 +33,26 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^11.2.14",
-    "@angular/cdk": "^11.2.13",
-    "@angular/common": "^11.2.14",
-    "@angular/compiler": "^11.2.14",
-    "@angular/core": "^11.2.14",
-    "@angular/flex-layout": "^11.0.0-beta.33",
-    "@angular/forms": "^11.2.14",
-    "@angular/material": "^11.2.13",
-    "@angular/platform-browser": "^11.2.14",
-    "@angular/platform-browser-dynamic": "^11.2.14",
-    "@angular/platform-server": "^11.2.14",
-    "@angular/router": "^11.2.14",
-    "@datorama/akita": "^6.1.3",
+    "@angular/animations": "^12.0.4",
+    "@angular/cdk": "^12.0.4",
+    "@angular/common": "^12.0.4",
+    "@angular/compiler": "^12.0.4",
+    "@angular/core": "^12.0.4",
+    "@angular/flex-layout": "^12.0.0-beta.34",
```

Comment:
Just looked, and this has been in beta for 4 years!

Reply:
:grimacing:

Reply:
yep

Reply:
On the other hand, gmail did five years https://slate.com/news-and-politics/2009/07/why-google-kept-gmail-in-beta-for-so-many-years.html